### ISSUE TITLE
Data.json agency submitted file snapshot

### ISSUE THREAD

**Issue by kvuppala** [GSA/datagov-wptheme#541]
During data.json harvest process, take a snapshot of the data.json that is harvested for archival purposes, this can be done on the manual reharvests + the regular automated harvests that happen daily/weekly

**Comment by rebeccawilliams** [comment]
+1

**Comment by hkdctol** [comment]
@kvuppala is this still an open item?

**Comment by kvuppala** [comment]
@hkdctol nope, but I think this snapshot is done by labs dashboard to a certain extent, not sure if this is still relevant in the catalog context.  Should we close this?

**Comment by hkdctol** [comment]
@kvuppala so there is equivalent through POD dashboard? If so we can close

**Comment by ksjustus** [comment]
This is Keedra Justus I have evidence on many people (some in police organizations). Please get with me. There are hidden cans in my home. I have even found illegal photos  of my little girl she is 6.  Dirty cops are involved. Please help us. 424 w Stumbo Perkins 4058860638. Please please help us. No one will help us
